exporter/lokiexporter/README.md

```diff
@@ -0,0 +1,62 @@
+# Loki Exporter
+
+Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/). By default, this exporter requires TLS and
+offers queued retry capabilities.
+
+Supported pipeline types: logs
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): The target URL to send Loki log streams to (
+  e.g.: https://loki.example.com:3100/loki/api/v1/push).
+
+
+- `allowed_attributes_for_stream_labels` (no default): List of allowed attributes to be added as labels to Loki log 
```

Comment:
This is so long even the example doesn't use it :) How about just `attributes_for_labels` or similar? Also should these be regexes for more flexibility?

Reply:
Absolutely agree on the length and the use of regular expressions. Updating now.